.github/config.yaml

```diff
@@ -0,0 +1,25 @@
+linkerdVersion: stable-2.8.0
```

Comment:
For the only reason that currently we want to skip the ingress tests in the CI - because KinD does not yet support Load balancers and having an external IP. Till we find a better alternative, we might have to have a config that skips ingress